ktor-server/ktor-server-plugins/ktor-server-sessions/api/ktor-server-sessions.api

```diff
@@ -85,6 +85,15 @@ public final class io/ktor/server/sessions/KotlinTimeJvmKt {
 	public static final fun setMaxAge-6Au4x4Y (Lio/ktor/server/sessions/CookieConfiguration;Lkotlin/time/Duration;)V
 }
 
+public final class io/ktor/server/sessions/KotlinxSessionSerializer : io/ktor/server/sessions/SessionSerializer {
```

Comment:
looks like this is something old, only changes in the end of the file should be there
need to just regenerate API dump?

Reply:
Yep, updating it. `apiDump` relies to much on gradle caches.